designer/client/src/components/AddProcessDialog.tsx

```diff
@@ -6,62 +6,46 @@ import { useProcessNameValidators } from "../containers/hooks/useProcessNameVali
 import HttpService from "../http/HttpService";
 import { WindowContent } from "../windowManager";
 import { AddProcessForm } from "./AddProcessForm";
-import { allValid, errorValidator } from "./graph/node-modal/editors/Validators";
+import { extendErrors, getValidationErrorsForField } from "./graph/node-modal/editors/Validators";
 import { useNavigate } from "react-router-dom";
+import { NodeValidationError } from "../types";
+import { isEmpty } from "lodash";
 
 interface AddProcessDialogProps extends WindowContentProps {
     isFragment?: boolean;
+    errors: NodeValidationError[];
 }
 
 export function AddProcessDialog(props: AddProcessDialogProps): JSX.Element {
-    const { isFragment, ...passProps } = props;
+    const { t } = useTranslation();
+    const { isFragment, errors = [], ...passProps } = props;
     const nameValidators = useProcessNameValidators();
 
     const [value, setState] = useState({ processId: "", processCategory: "" });
-    const [processNameError, setProcessNameError] = useState({
-        fieldName: "processName",
-        message: "",
-        description: "",
-        typ: "",
-    });
 
-    const isValid = useMemo(() => value.processCategory && allValid(nameValidators, [value.processId]), [nameValidators, value]);
+    const fieldErrors = getValidationErrorsForField(extendErrors(errors, value.processId, "processName", nameValidators), "processName");
 
     const navigate = useNavigate();
     const createProcess = useCallback(async () => {
-        if (isValid) {
+        if (isEmpty(fieldErrors)) {
             const { processId, processCategory } = value;
-            try {
-                await HttpService.createProcess(processId, processCategory, isFragment);
-                passProps.close();
-                navigate(visualizationUrl(processId));
-            } catch (error) {
-                if (error?.response?.status == 400) {
-                    //TODO: change to pass error from BE as whole object not just the message
-                    setProcessNameError({ fieldName: "processName", message: error?.response?.data, description: "", typ: "" });
-                } else {
-                    throw error;
-                }
-            }
+            await HttpService.createProcess(processId, processCategory, isFragment);
```

Comment:
I removed them intentionally, because my assumption was, that it would just duplicate frontend errors. Now we check the uniqueness and mandatory of the field name. Are there more cases on the backend to return errors?

Reply:
BE additionally validates scenario/fragment id checking for: trailing, leading spaces, blank values and we have engine-specific custom validations like for flink - returning `Only digits, letters, underscore (_), hyphen (-) and space in the middle are allowed`.

Reply:
Thanks for the explanation, changed ✔️